spikeinterface/sorters/runsorter.py

```diff
@@ -482,16 +476,18 @@ def run_sorter_container(
     if verbose:
         print('Starting container')
     container_client.start()
-    print("******")
-    print("Container started with the following paths")
-    print(si_dev_path_unix, si_source_folder)
+    if verbose:
+        print("******")
+        print("Container started with the following paths")
+        print(si_dev_path_unix, si_source_folder)
+    
     # check if container contains spikeinterface already
     cmd_1 = ['python', '-c', 'import spikeinterface; print(spikeinterface.__version__)']
     cmd_2 = ['python', '-c', 'from spikeinterface.sorters import run_sorter_local']
     res_output = ''
     for cmd in [cmd_1, cmd_2]:
         res_output += str(container_client.run_command(cmd))
-    need_si_install = 'ModuleNotFoundError' in res_output or running_on_github_actions
+    need_si_install = 'ModuleNotFoundError' in res_output
```

Comment:
This one, some of the sorters already have spikeinterface installed. Would it be useful to know if they still run with the latest version of spikeinterface. The option was kind of forcing the installation of the latest version of spikeinterface.

Reply:
By construction they don't :) this is because so we make sure that we are running the correct SI version in the container. The only option to have SI installed is to manually extend the container. In that case, since it does require a fair amount of actions, I think that the user wants to use the version s/he installed

Reply:
You mean that by construction they don't have spikeinteface installed?

In that case, the `need_si_install` will always be True? (and therefore the force install is superflous?).

Or am I misreading?

Reply:
The containers that we ship are without spikeinterface. However a user could make a docker container (say `my-docker-id/my-container` with the following Dockerfile:

```
FROM spikinterface/kilosort2-compiled-base:latest

pip install spikeinterface==0.97
```

If the user runs now `ss.run_sorter("kilosort2", docker_image=my-docker-id/my-container:latest")`, SpikeInterface is installed and explicitly pinned to a certain version, so we don't reinstall. If something fails, it's not our fault ;)

Reply:
OK, thanks for explaining it to me.

But that means that for all the sorters that we are testing here in the CI SHOULD install spikeinterface? I am asking because when I was testing this locally I added that bool because some the sorters were not installing spikeinterface. Maybe it was a problem on my machine only.

After we merge this we can verify this.